GDJS/Runtime/pixi-renderers/pixi-bitmapfont-manager.ts

```diff
@@ -316,19 +316,27 @@ namespace gdjs {
     }
 
     unloadResource(resourceData: ResourceData): void {
-      const loadedFont = this._loadedFontsData.getFromName(resourceData.name);
-      if (loadedFont) {
-        this._loadedFontsData.delete(resourceData);
-      }
+      this._loadedFontsData.delete(resourceData);
 
       for (const bitmapFontInstallKey in this._pixiBitmapFontsInUse) {
-        if (bitmapFontInstallKey.endsWith(resourceData.file))
+        if (bitmapFontInstallKey.startsWith(resourceData.name)) {
```

Comment:
I think that should be 
```suggestion
        if (bitmapFontInstallKey.startsWith(resourceData.name + '@')) {
```

to avoid issues when a resource name is a prefix of another (not that unlikely, it can happen if a resource name is "MyBitmapText" and another is "MyBitmapTextSpecial" or "MyBitmapText2", etc...)